doc/ref_model/chapters/chapter09.md

```diff
@@ -243,3 +247,82 @@ auto.cicd.014 | The CD pipeline must be able to automatically promote software f
+
+
+<a name="9.6.1"></a>
+### 9.6.1. Why Observability
+
+Knowing the status of all services and functions at all levels in a cloud based service offering is essential to act fast, ideally pro-actively before users notice and, most importantly, before they call the help desk. This requires the collection of alarms and telemetry data from the physical layer (wires), the cloud infrastructure up to the network, application and services virtualized functions (VNF) running on top of Cloud Infrastructure, typically isolated by tenants.
+
+Long term trending data are essential for capacity planning purposes and typically collected, aggregated and kept over the full lifespan. To keep the amount of data collected manageable, automatic data reduction algorithms are typically used, e.g. by merging data points from the smallest intervals to more granular intervals.
+
+A cloud typically consists of one or more regional data centers, central offices, and edge sites. These are managed from redundant central management sites, each hosted in their own data centers.
+
+While many Telco Clouds start as a vertical cloud by hosting one prime application, e.g. IMS, the intent is to host more applications over time in order to truly maximize the cloud to its full potential. Therefore it is pure coincidence to have the same team responsible and manage the infrastructure and the application running on top. Network services and applications deployed on a Telco Cloud are managed by separate teams, within the same or different organizations and a monitoring solution must be capable of keeping the collection of monitoring data isolated between tenants and NFVI. At the same time, some monitoring data from the NFVI layer must selectively be available to tenant monitoring applications in order to correlate e.g. VNF metrics with the underlying infrastructure it currently runs on.
+
+<a name="9.6.2"></a>
+### 9.6.2. What to Monitor
+
+These physical and virtual devices need to be monitored: 
+
+* Network Services across sites and tenants 							
+* Virtualized functions per site and tenant
+* Individual Virtual Machines and Containers
+* Virtualization infrastructure components
+* Physical servers (compute) and network elements
+* Toolservers with their applications (DNS, IdM, ZTP, etc) 
```

Comment:
"IdM, ZTP" -> "Identity Management, Zero Touch Provisioning".

Reply:
@petorre I have updated the text per your suggestion